blank_rust_project/package.json

```diff
@@ -1,32 +1,32 @@
 {
-    "name": "awesome-project-rust",
-    "description": "A full stack project using rust smart contract on NEAR",
-    "version": "0.0.1",
-    "scripts": {
-        "build": "cd contract && ./build.sh",
-        "dev:deploy:contract": "near dev-deploy --wasmFile ./contract/res/status_message.wasm",
-        "deploy:contract": "near deploy --wasmFile ./contract/res/status_message.wasm",
-        "deploy:pages": "gh-pages -d src",
-        "deploy": "npm run build && npm run deploy:contract && npm run deploy:pages",
-        "prestart": "npm run build && near deploy --wasmFile ./contract/res/status_message.wasm",
-        "start": "serve src",
-        "test": "cd contract && cargo test --package status-message -- --nocapture"
-    },
-    "devDependencies": {
-        "gh-pages": "^2.2.0",
-        "gulp": "^4.0.2",
-        "jest": "^24.9.0",
-        "jest-environment-node": "^24.9.0",
-        "near-runtime-ts": "^0.5.2",
-        "near-shell": "^0.20.1",
-        "near-bindgen-as": "^1.2.0"
-    },
-    "wasmStudio": {
-        "name": "Hello World Example",
-        "description": "The contract implements a single function to return \"Hello, World!\" using AssemblyScript",
-        "icon": "typescript-lang-file-icon"
-    },
-    "dependencies": {
-        "serve": "^11.3.0"
-    }
+  "name": "near-blank-project",
+  "version": "0.0.1",
+  "scripts": {
+    "build": "npm run build:contract && npm run build:web",
+    "build:contract": "cd contract && ./build.sh",
+    "build:web": "parcel build src/index.html --public-url ./",
+    "dev:deploy:contract": "near dev-deploy --wasmFile ./contract/res/status_message.wasm",
+    "deploy:contract": "near deploy --wasmFile ./contract/res/status_message.wasm",
+    "deploy:pages": "gh-pages -d dist/",
+    "deploy": "npm run build && npm run deploy:contract && npm run deploy:pages",
+    "prestart": "npm run build:contract && npm run dev:deploy:contract",
+    "start": "CONTRACT_NAME=$(cat neardev/dev-account) parcel src/index.html",
+    "dev": "nodemon --watch assembly -e ts --exec 'npm run start'",
+    "test": "cd contract && cargo test --package status-message -- --nocapture"
+  },
+  "devDependencies": {
+    "gh-pages": "^2.2.0",
+    "gulp": "^4.0.2",
+    "jest": "^25.1.0",
+    "jest-environment-node": "^24.9.0",
+    "near-bindgen-as": "^1.2.0",
+    "near-runtime-ts": "^0.5.2",
+    "near-shell": "^0.20.1",
+    "nodemon": "^2.0.2",
+    "parcel-bundler": "^1.12.4"
+  },
+  "dependencies": {
+    "nearlib": "^0.21.0",
+    "regenerator-runtime": "^0.13.3"
```

Comment:
what is regenerator-runtime used for?

Reply:
it's used to support `async` / `await` in browsers which don't support it natively.

Reply:
or to be more precise for browsers without support for generators, which are somewhat related concept which can be used to fake `async` / `await`